site/src/pages/WorkspacesPage/WorkspacesPageView.tsx

```diff
@@ -34,93 +33,77 @@ export interface WorkspacesPageViewProps {
   error?: unknown
 }
 
-export const WorkspacesPageView: FC<WorkspacesPageViewProps> = (props) => {
-  const styles = useStyles()
+export const WorkspacesPageView: FC<WorkspacesPageViewProps> = ({ loading, workspaces }) => {
```

Comment:
the `error` prop isn't used -> can we just remove it from the `WorkspacesPageViewProps` interface then?